<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverAid</class>
 <widget class="QMainWindow" name="OverAid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1650</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OverAid</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_filtre">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="title">
       <string>Filtres</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item row="0" column="4" rowspan="2">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="11">
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="7">
             <widget class="QDateEdit" name="dateEditFilter_start">
              <property name="dateTime">
               <datetime>
                <hour>8</hour>
                <minute>3</minute>
                <second>54</second>
                <year>1900</year>
                <month>1</month>
                <day>1</day>
               </datetime>
              </property>
              <property name="time">
               <time>
                <hour>8</hour>
                <minute>3</minute>
                <second>54</second>
               </time>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Dépense / Entrée :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="9">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Devise :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QPushButton" name="pushButtonFiltre_cat2">
              <property name="text">
               <string>Sous-catégories</string>
              </property>
             </widget>
            </item>
            <item row="1" column="6">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Date de fin :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="12">
             <widget class="QPushButton" name="pushButtonFiltre_valide">
              <property name="text">
               <string>✓</string>
              </property>
             </widget>
            </item>
            <item row="1" column="7">
             <widget class="QDateEdit" name="dateEditFilter_end">
              <property name="dateTime">
               <datetime>
                <hour>18</hour>
                <minute>0</minute>
                <second>0</second>
                <year>2100</year>
                <month>12</month>
                <day>30</day>
               </datetime>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="12">
             <widget class="QPushButton" name="pushButtonFiltre_delete">
              <property name="text">
               <string>✕</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="comboBoxFiltre_inOut">
              <item>
               <property name="text">
                <string>Tout</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dépense</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Entrée d'argent</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Date de début :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QComboBox" name="comboBoxFiltre_moyen">
              <property name="placeholderText">
               <string/>
              </property>
              <item>
               <property name="text">
                <string>Tout</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Carte bancaire</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Éspèces</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Chèque</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Virement</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Prélèvement</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="pushButtonFiltre_cat">
              <property name="text">
               <string>Catégories</string>
              </property>
             </widget>
            </item>
            <item row="1" column="10">
             <widget class="QComboBox" name="comboBoxFiltre_devise"/>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Moyen de paiement :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="8">
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="9" colspan="2">
             <widget class="QLineEdit" name="lineEditFiltre_desc">
              <property name="whatsThis">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;'%' remplace une chaîne de caractère&lt;/p&gt;&lt;p&gt;'_' remplace un seul caractère&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="placeholderText">
               <string>Description</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3" rowspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_savedFilters"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Filtres sauvegardés :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_idSavedFilter">
         <property name="text">
          <string>id_filtre</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>400</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabAccount">
       <attribute name="title">
        <string>Transactions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="2" column="2">
         <widget class="QLabel" name="label_prevision">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Prévision compte :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_reste">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Reste compte : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QTreeWidget" name="treeWidgetSummary">
          <property name="minimumSize">
           <size>
            <width>750</width>
            <height>0</height>
           </size>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <attribute name="headerCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0" rowspan="3">
         <widget class="QWidget" name="widgetTrans" native="true">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStats">
       <attribute name="title">
        <string>Statistiques</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelStats_date">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelStats_montant">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_stats"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1650</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuG_rer">
    <property name="title">
     <string>Gérer</string>
    </property>
    <widget class="QMenu" name="menuComptes">
     <property name="title">
      <string>Comptes</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionAjouter_un_compte"/>
    </widget>
    <widget class="QMenu" name="menuLangage">
     <property name="title">
      <string>Langage</string>
     </property>
     <addaction name="actionFran_ais"/>
     <addaction name="actionAnglais"/>
     <addaction name="actionEspagnol"/>
    </widget>
    <addaction name="menuComptes"/>
    <addaction name="menuLangage"/>
    <addaction name="separator"/>
    <addaction name="actionCat_gories"/>
    <addaction name="actionAbonnements"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menufirst">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionExporter_au_format_CSV"/>
    <addaction name="actionExporter_les_PDF_en_ZIP"/>
    <addaction name="actionSauvegarder_la_base_de_donn_es"/>
    <addaction name="separator"/>
    <addaction name="actionFermer"/>
   </widget>
   <addaction name="menufirst"/>
   <addaction name="menuG_rer"/>
  </widget>
  <action name="actionAjouter_un_compte">
   <property name="text">
    <string>Gérer les comptes</string>
   </property>
  </action>
  <action name="actionCat_gories">
   <property name="text">
    <string>Catégories</string>
   </property>
  </action>
  <action name="actionFran_ais">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/qrc/ressources/image/flag_france.png</normaloff>:/qrc/ressources/image/flag_france.png</iconset>
   </property>
   <property name="text">
    <string>Français</string>
   </property>
  </action>
  <action name="actionAnglais">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/qrc/ressources/image/flag_usa.png</normaloff>:/qrc/ressources/image/flag_usa.png</iconset>
   </property>
   <property name="text">
    <string>Anglais</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionExporter_au_format_CSV">
   <property name="text">
    <string>Exporter au format CSV</string>
   </property>
  </action>
  <action name="actionSauvegarder_la_base_de_donn_es">
   <property name="text">
    <string>Sauvegarder la base de données</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExporter_les_PDF_en_ZIP">
   <property name="text">
    <string>Télécharger tous les PDF</string>
   </property>
  </action>
  <action name="actionAbonnements">
   <property name="text">
    <string>Abonnements</string>
   </property>
  </action>
  <action name="actionEspagnol">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/qrc/ressources/image/flag_spain.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Espagnol</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>treeWidgetSummary</tabstop>
 </tabstops>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
